addons/controllers/csi/vspherecsiconfig_utils.go

```diff
@@ -26,6 +30,56 @@ import (
 	topologyv1alpha1 "github.com/vmware-tanzu/vm-operator/external/tanzu-topology/api/v1alpha1"
 )
 
+// ClusterToVSphereCSIConfig returns a list of Requests with VSphereCSIConfig ObjectKey
+func (r *VSphereCSIConfigReconciler) ClusterToVSphereCSIConfig(o client.Object) []ctrl.Request {
```

Comment:
nit: maybe rename to `VSphereClusterToVSphereCSIConfig` like https://github.com/vmware-tanzu/tanzu-framework/blob/main/addons/controllers/cpi/vspherecpiconfig_utils.go#L34. But I'm ok with the name